src/components/ApplicationBorrowerTable.tsx

```diff
@@ -141,6 +141,19 @@ export function ApplicationBorrowerTable({
               formatter={renderSector}
               modifiedFields={application.modified_data_fields?.borrower_updates}
             />
+            <ApplicationTableDataBorrowerRow
+              isLoading={isLoading}
```

Comment:
I don't know anything about the frontend, but the other rows do `isLoading={isLoading || isLoadingVerifyDataField}`. I think maybe this was adapted from `ApplicationAwardTable` but this property was not updated for `ApplicationBorrowerTable`?